lighthouse-cli/test/fixtures/dobetterweb/bad-promise-polyfill.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/**
+ * @fileoverview
+ * Lighthouse backstory:
+ * Zone aggressively polyfills promises including rewriting Promise.prototype.then which caused
+ * problems for our evaluateAsync code where we rely on a window.__nativePromise to work.
+ * We serve this file in dbw_tester.html to make sure evaluateAsync, etc all works great
+ * Serving this file via node_modules was a pain so we've vendored it here.
```

Comment:
```suggestion
 * Serving this file via node_modules was a pain so we've replaced it with our own.
```

right? or is this actually a copy of zone.js?